libs/cardano-ledger-binary/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Version history for `cardano-ledger-binary`
 
-## 1.4.0.1
+## 1.4.1.0
```

Comment:
Sorry, I did not catch it earlier, but adding a new constructor to a sum type is commonly considered a breaking change.
```suggestion
## 1.5.0.0
```